tools/validators/ontology_validator/yamlformat/validator/external_file_lib.py

```diff
@@ -85,3 +85,16 @@ def _RecursiveDirWalk(directory):
           base_lib.PathParts(root=directory, relative_path=relative_path_yaml))
 
   return path_parts
+
+def RecursiveDirWalkPublic(directory):
```

Comment:
no need to define a new method

Reply:
`gpylint` complains that I'm calling a private method if I call `_RecursiveDirWalk`, so I made this public-facing method

Reply:
just remove the `_` which by convention is for private methods

Reply:
The function `RecursiveDirWalk` doesn't exist? So when I remove the `_` it throws `AttributeError: module 'yamlformat.validator.external_file_lib' has no attribute 'RecursiveDirWalk'`.

Reply:
Done